src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/AbstractDocumentSpanEntry.cs

```diff
@@ -45,31 +50,56 @@ private abstract class AbstractDocumentSpanEntry : Entry
             }
 
             protected StreamingFindUsagesPresenter Presenter => _context.Presenter;
+            protected SourceText OriginalSourceText => _sourceText;
+
 
-            protected Document Document => _documentSpan.Document;
-            protected TextSpan SourceSpan => _documentSpan.SourceSpan;
+            protected Document Document => GetMappedPosition().Document;
+            protected TextSpan SourceSpan => SourceText.Lines.GetTextSpan(GetMappedPosition().Span);
```

Comment:
we should just have Span, but now I am using legacy API to convert secondary buffer to primary buffer, that is only reason Span is line/char. once move to correct, IProjectBuffer API, Span should be just there. no need to convert.